deploy/terraform-datacommons-website/modules/bt_cache/variables.tf

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+variable "resource_suffix" {
+  type        = string
+  description = "Optional suffix to add to GCP resources for uniqueness."
+  default     = null
+}
+
+variable "project_id" {
+  type        = string
+  description = "This is the same GCP project id from the setup step."
+}
+
+variable "region" {
+  type        = string
+  description = "Region to setup the Cloud Function in. Should match the region of the other resources."
+}
+
+variable "dc_resource_bucket" {
+  type        = string
+  description = "Name of the GCS bucket that stores DC resources. Do not include the scheme(gs://)."
+}
+
+variable "csv2bt_template_path" {
+  type        = string
+  description = "Full GCS path, including scheme, to the Dataflow template to load BT cache."
+  default     = "gs://datcom-dataflow-templates/templates/flex/csv_to_bt.json"
```

Comment:
This will be created from the commands in this PR: https://github.com/datacommonsorg/tools/pull/163.